barretenberg/cpp/.clangd

```diff
@@ -1,78 +0,0 @@
-CompileFlags: # Tweak the parse settings
-  Remove: -fconstexpr-ops-limit=*
```

Comment:
Accident?

Reply:
Definitely, I don't even know what that is. Thanks for flagging!

Reply:
Actually, [it is supposed to be ignored ](https://github.com/AztecProtocol/aztec-packages/blob/533b83e88e8f91d60aa3e59a174b001272105698/barretenberg/cpp/.gitignore#L10). I did clean up my working tree with `git rm -r --cached .` so it could have been just hanging in the repo despite being supposed to be ignored.

Not sure what to do, but I'd actually merge. Not sure how can i even "un-ignore" it now

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/4201 We definitely use that file. Let's fix the gitignore

Reply:
Coll it will show back up when I rebase, thanks!